src/BlazorStrap/Modal.razor

```diff
@@ -48,7 +53,7 @@
         }
     }
     private bool _isOpen;
-    [Parameter]
```

Comment:
I see you added Hide/Show and removed IsOpen which makes sense, but I think we should still leave IsOpen as a Parameter and allow both styles. I can add it back in before I merge, just wanted to know if you see any problems with it.

Reply:
Thought I added it back in was going to remove it then thought better of breaking peoples code

Reply:
No worries, adding it back, I also need to disable the Esc key when "keyboard: false" (or a better name) per #81 as mentioned here: https://getbootstrap.com/docs/4.3/components/modal/#options and then I will send out the new release. This PR is really great! Much better than the "alert" I had in SSB project :)

Reply:
Oops! You already added the IgnoreEscape! Sorry about that, didnt look before I wrote the above

Reply:
I wanted to include the static files in the Core project but not really sure how to do it with client-side

Reply:
At some point I heard someone has a project that does that. I will try to find it